README.md

```diff
@@ -0,0 +1,320 @@
+
+# Cat API #
+
+# A REST API using ExpressJS#
+
+A data repository to manage information about cats and breeds via the HTTP request-response cycle.
+
+# Getting Started #
+
+Run 'npm start' on Terminal.
```

Comment:
Also requires `npm i` on a fresh checkout. Might also be worth mentioning that `npm test` runs the Jest/Supertest test cases

Reply:
This is the same as npm install, is that correct?

Reply:
Yep. You can run `npm help` - most programs have an instruction manual which list all the possible arguments.